guide/event-driven-and-pooling-consumer-service/tests/event-driven-and-pooling-consumer_test.bal

```diff
@@ -0,0 +1,173 @@
+@http:ServiceConfig {
+    basePath: "/"
+}
+service<http:Service> SensorEventService bind sensorEventListner {
+    @http:ResourceConfig {
+        methods: ["POST"], consumes: ["application/json"], produces: ["application/json"],
+        path: "/activity"
+    }
+    documentation { Resources for reciving geo activities captured from the sensor }
+    activity(endpoint conn, http:Request req) {
+        json requestMessage = check req.getJsonPayload();
+        requestmessage1 = untaint requestMessage.toString();
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/health"
+    }
+    documentation { Resource for checking the health of the EP of the sensor}
+    health(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        requestmessage2 = untaint requestMessage.toString();
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/maintenance"
+    }
+    documentation { Resource for invoking a service when the sensor needed mainteinance }
+    maintanance(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        requestmessage3 = untaint requestMessage.toString();
+        res.setPayload("Ack - maintenance team on the way");
+    }
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/calibrate"
+    }
+    documentation { Resource for invoking a service when the sensor needed calibration }
+    calibrate(endpoint conn, http:Request req) {
+        http:Response res = new;
+        json requestMessage = check req.getJsonPayload();
+        requestmessage4 = untaint requestMessage.toString();
+        res.setPayload("Ack - Calibration scheduled");
+    }
+}
+function messageSender() {
+    http:Request req1;
+    json sampleRequest1 = { "ID": 27125088, "SID": 4344, "TIME_S": "1536423224", "TIME_E": "1536423584", "TYPE": "EQ",
+        "C_DATA": "41.40338, 2.17403", "R_SCALE": 10 };
+    string msg1 = sampleRequest1.toString();
+    mb:Message message1 = check queuesensorGeoMessage.createTextMessage(msg1);
+    _ = queuesensorGeoMessage->send(message1);
+    http:Request req2;
+    json sampleRequest2 = { "ID": 67602894, "SID": 1781, "TIME_S": "1536596384", "STATUS": "1122" };
+    string msg2 = sampleRequest2.toString();
+    mb:Message message2 = check queuesensorGeoMessage.createTextMessage(msg2);
+    _ = queueHealthCheck->send(message2);
+    http:Request req3;
+    json sampleRequest3 = { "ID": 88885089, "SID": 5848, "TIME_S": "1536578384", "DATA":
+    "ROTC1234 module need to be replaced" };
+    string msg3 = sampleRequest3.toString();
+    mb:Message message3 = check queuesensorGeoMessage.createTextMessage(msg3);
+    _ = queueMaintenance->send(message3);
+    http:Request req4;
+    json sampleRequest4 = { "ID": 54256677, "SID": 7098, "TIME_S": "1536599984", "DATA": "Sensor need to be calibrated"
```

Comment:
Need to reformat.

Reply:
Reformat all the files before the last commiy